src/QMCWaveFunctions/tests/test_MO.cpp

```diff
@@ -144,6 +144,57 @@ void test_He(bool transform)
     REQUIRE(dpsi[0][2] == Approx(0));
     REQUIRE(d2psi[0] == Approx(-0.2618497452));
 
+    //==========Hessian and Grad Hessian Test==========
+    /*
+    // Generated from gen_mo.py for position [0.1, 0.0, 0.0]
+    REQUIRE(values[0] == Approx(   0.9671159429));
```

Comment:
Why are these tests commented?

Reply:
I generated the test data at the beginning of the project, but as I went on, I felt like the HCN example was a lot more thorough/discriminatory than the Ne or He tests, so I didn't enable the tests for those.

Reply:
Does it hurt anything if we enable tests?
I think it is better to have them enabled or deleted not commented.

Reply:
It doesn't hurt anything except add quite a bit to the source. I might delete these, since I imagine I have other more important tests to add soon.